transcom/mymove
# [MB-14736] Update GEX URL in all non-prod environments

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14736) for this change

## Summary

This PR updates the URL for GEX in all environments except for prod (which would come later).

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

## Verification Steps for Author

These are to be checked by the author.


## Verification Steps for Reviewers

Just make sure I didn't miss any relevant usages of the URL that need changing.  I read that [we're only able to connect to GEX from staging and prod](https://dp3.atlassian.net/wiki/spaces/MT/pages/1775796264/GEX+Account+Password+Rotation+and+Management), so I'm not sure that we'll be able to tell if it works or not until we merge and look at the staging logs.

Update: I was able to use `post-file-to-gex` [via the instructions here](https://docs.google.com/document/d/180EI6KRVTqlFPDGqIpCt4_zajs2zSqLi/edit#) (after setting up the local GEX_URL in my `.envrc.local`) and the new URL seemed to work.
